app/assets/stylesheets/articles.scss

```diff
@@ -0,0 +1,28 @@
+#articles-header {
+    text-align: center;
+}
+
+#articles-details {
+    border: black 2px solid;
+    margin: 0 0 20px 0;
+    background-color: rgba(0,0,0,.5);
+
+    #articles-title {
+        text-decoration: underline wavy black;
+    }
+}
+
+#articles-details > * {
+    padding: 10px;
+}
+
+#articles-details-info {
+    display: flex;
+    flex-flow: row wrap;
+    background-color: white;
+
+    p {
+        margin: 0 20px 0 0;
+        padding: -20px;
+    }
+} 
```

Comment:
These two could be nested under `#articles-details`. The first one can be added as `> * {...` under the `#articles-details` group. The second one can just be copy pasted.

Reply:
SCSS nesting is qwazy